module/third_party/proto/tendermint/types/validator.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+package tendermint.types;
+
+option go_package = "github.com/tendermint/tendermint/proto/tendermint/types";
+
+import "gogoproto/gogo.proto";
+import "tendermint/crypto/keys.proto";
```

Comment:
These are not strictly necessary for our proto generation, since they are not dependencies our protos directly use. They are indirect dependencies which are needed for the proto linting to pass.